ignite/metrics/metric.py

```diff
@@ -290,8 +293,8 @@ def compute(self):
 
             res = evaluator.run(data)
 
-    .. versionchanged:: 0.4.2
```

Comment:
Actually, we need to keep this and add the new `versionchanged` tag:
```
.. versionchanged:: 0.4.2
    ``required_output_keys`` became public attribute.

.. versionchanged:: 0.5.1
    ``skip_unrolling`` attribute is added.
```
So, let's revert this suppression